scripts/enst_to_uniprot_mapping.py

```diff
@@ -0,0 +1,196 @@
+# 1. Mapping by protein sequence: ENST -> ENSP -> Ensembl sequence = UniProt sequence -> UniProt id
+# Sequence from Ensembl:
+# GRCh37: http://ftp.ensembl.org/pub/grch37/release-101/fasta/homo_sapiens/pep/ rename to ensembl_grch37.fa.gz
+# GRCh38: http://ftp.ensembl.org/pub/release-101/fasta/homo_sapiens/pep/ rename to ensembl_grch38.fa.gz
+# Sequence from UniProt: https://www.uniprot.org/uniprot/?query=+reviewed%3Ayes+AND+organism%3A%22Homo+sapiens+%28Human%29+%5B9606%5D%22&sort=score
+# 2. handle multiple UniProt cases
+# Biomart mapping
+# GRCh37: https://grch37.ensembl.org/biomart/martview/145410cb02e693b9da427f9c3dffe61f
+# GRCh38: http://uswest.ensembl.org/biomart/martview/84262bd39b9c7d095757b6ce493a2a9f
+# Reviewed map:
+# GRCh37: https://docs.google.com/spreadsheets/d/14PN6RtFq_GTAu8OKNUyUNKJ_fj7OlcEhDjbMdapqqo8/edit#gid=133083142
+# GRCh38: https://docs.google.com/spreadsheets/d/1slDx9zorUuA-xsmH1i9_6CIGjQB1GIgDlWDU5gw6f9I/edit#gid=1399879714
+# Output:
+# "grch3x_ensemblxx_enst_to_uniprot_mapping_full.txt" is the full spreadsheet generated by the script, it contains all review process columns
+# "grch3x_ensemblxx_enst_to_uniprot_mapping_id.txt" is the mapping file with only enst column and uniprot mapping column
+
+import json
+import re
+import pandas as pd
+import numpy as np
+from Bio import SeqIO
+import wget
+import requests
+from io import StringIO
+import time
+from collections import OrderedDict
+import argparse
+import subprocess
+
+# generate sequence to uniprot id dictionary
+def generate_dict(key, value, dictionary):
+    if key not in dictionary:
+        dictionary[key] = []
+    dictionary[key].append(value)
+def generate_uniprot_protein_length(ensp, dictionary, sequence_to_uniprot_dict):
```

Comment:
add whitespace before the function def?
```suggestion

def generate_uniprot_protein_length(ensp, dictionary, sequence_to_uniprot_dict):
```